code/drasil-website/lib/Drasil/Website/About.hs

```diff
@@ -99,6 +105,14 @@ aboutParagraph8 paperGOOL = S "As described in the" +:+ namedRef paperGOOL (S "G
   \generation, allowing Drasil to more efficiently generate code in several languages, including Python, Java, C-Sharp, and C++."
 
 -- | Paragraph providing a link to Drasil papers and documents
-aboutParagraph9 :: Reference -> Sentence
-aboutParagraph9 papersWiki = S "A list of papers and documents written about Drasil can be found in the" +:+ namedRef papersWiki (S "Drasil Papers \
-  \and Documents") +:+ S "wiki page"
\ No newline at end of file
+aboutParagraph9 :: Reference -> Reference -> Reference -> Reference -> Sentence
+aboutParagraph9 papersWiki icsePositionPaper danPoster wellUnderstoodPaper = 
+  S "A list of papers and documents written about Drasil can be found on the" +:+
+  namedRef papersWiki (S "Drasil papers and documents") +:+. wikiSentence +:+
```

Comment:
I previously asked if "papers" and "documents" should be capitalized; I now realize that "Drasil Papers and Documents" is the name of the [wiki page](https://github.com/JacquesCarette/Drasil/wiki/Drasil-Papers-and-Documents) being referenced here, so they should be capitalized. Other than this, this PR looks good to me!